### PR TITLE
Fix non-ascii characters in test/e2e/common/runtime.go

### PR DESCRIPTION
**Pull request by YuikoTakada**
**What type of PR is this?**
/kind bug
/kind cleanup

**What this PR does / why we need it**:
Fix non-ascii characters which are written accidentally.

**Which issue(s) this PR fixes**:

Ref #87679

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/common/runtime.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ var _ = framework.KubeDescribe("Container Runtime", func() {
 			/*
 				Release : v1.13
 				Testname: Container Runtime, Restart Policy, Pod Phases
-				Description: If the restart policy is set to ‘Always’, Pod MUST be restarted when terminated, If restart policy is ‘OnFailure’, Pod MUST be started only if it is terminated with non-zero exit code. If the restart policy is ‘Never’, Pod MUST never be restarted. All these three test cases MUST verify the restart counts accordingly.
+				Description: If the restart policy is set to 'Always', Pod MUST be restarted when terminated, If restart policy is 'OnFailure', Pod MUST be started only if it is terminated with non-zero exit code. If the restart policy is 'Never', Pod MUST never be restarted. All these three test cases MUST verify the restart counts accordingly.
```
Comment (liggitt) [inline test/e2e/common/runtime.go]: this requires changing the conformance.txt file as well... see the verify CI job failure

Comment (YuikoTakada) [reply test/e2e/common/runtime.go]: thank you for your comment. sorry, I can't find error messages about conformance.txt in https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/87680/pull-kubernetes-verify/1222686544863694851 could you please show me the point?

Comment (liggitt) [reply test/e2e/common/runtime.go]: sorry, I was wrong... the description isn't pulled into test/conformance/testdata/conformance.txt

**Review by feiskyer — COMMENTED** [review]
/retest
/priority important-soon

**Pull request merged**